packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -438,7 +439,15 @@ export class EditManager<
 			0x428 /* Clients with local changes cannot be used to generate summaries */,
 		);
 
-		const trunk = getPathFromBase(this.trunk.getHead(), this.trunkBase).map((c) => {
+		const [_, oldestCommitInCollabWindow] = this.getClosestTrunkCommit(
+			this.minimumSequenceNumber,
+		);
+
+		const trunk = getPathFromBase(
+			this.trunk.getHead(),
+			// Path construction is exclusive, so we need to use the parent of the oldest commit in the window if it exists
+			oldestCommitInCollabWindow.parent ?? oldestCommitInCollabWindow,
```

Comment:
If the oldestCommitInCollabWindow does not have a parent, then `oldestCommitInCollabWindow === this.trunkBase`, correct? I think it might be helpful to mention that in the comment, or put an `assert(oldestCommitInCollabWindow === this.trunkBase)` in that case so that it's more self-documenting as to when this scenario happens (it happens only when the collab window is the whole trunk).

Reply:
Added an assert!